lib/versioned/runner.js

```diff
@@ -97,7 +97,12 @@ function installPackages(cb) {
 function runTests(cb) {
   // TODO: Add tap arguments, such as color.
   process.send({ status: 'running' })
-  spawn('node', [testFile], function testHandler(err) {
+  let args = [testFile]
+  if (process.env.packageType && process.env.packageType === 'module') {
+    args = ['--loader ../../../../nr-esm-loader/test-loader.mjs', testFile]
```

Comment:
i'm thinking we allow this to be changed via env var.  also right now this test loader lives in `/test/lib/test-loader.mjs`in agent repo. But i could see this loader moving to this repo or having bespoke ones for external repos

Reply:
also not sure but should we `path.resolve` on the path passed in? at the very least that could just be `newrelic/test/lib/test-loader.mjs` right?

Reply:
Yeah--I'd be in favor of both: an env var to pass it in, but a default/fallback that uses `path.resolve`.

Reply:
[The most recent module loader PR](https://github.com/newrelic/node-newrelic/pull/1348/files) has the loader at the agent's root level. If we want to have a separate test loader, I could see targeting that...but for now this seems to be the canonical loader location.